sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -3359,6 +3360,31 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       enddo
 !     if(debugprint)print*,'sample l',VarName,' = ',1,isltyp(isa,jsa)
       
+      IF(MODELNAME == 'FV3R')THEN
```

Comment:
@EricJames-NOAA The subroutine netcdf_2d_para will set the variable buf as spval if it can't read in variable "wet1" from model file. So I would think the IF condition is not necessary. You might only add the changes of reading "wet1" here(line 3364-3372).

Reply:
@WenMeng-NOAA thanks!  Good point.  I made that change.